tests/workspace/test_manager.py

```diff
@@ -86,7 +86,7 @@ def test_bundle_of_invalid_entity_type(temp_dir):
 
 @pytest.mark.parametrize(
     "project_directory_name",
-    ["migration_streamlit_V1_to_V2", "migration_snowpark_V1_to_V2"],
+    ["migration_streamlit_v1_to_v2", "migration_snowpark_V1_to_V2"],
```

Comment:
Why changed only in streamlit?

Reply:
ruff was having problems with this name (don't ask me why) - https://github.com/snowflakedb/snowflake-cli/actions/runs/10618285380/job/29433633082

Reply:
So could you change V->v in snowpark too?

Reply:
Done